src/current/advisories/a131639.md

```diff
@@ -12,9 +12,11 @@ Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
 
 ## Description
 
-In v22.2 through v24.2, in circumstances where there is a sustained period of disk slowness in the presence of lease transfers, it is possible for some writes in a transaction that straddle multiple ranges to be lost. In cases where these lost writes are not also present on one or more secondary indexes that could allow for reconstructing or repairing the row, these writes could be irrecoverably lost.
+In v22.2 through v24.2, in rare circumstances where there is a sustained period of disk slowness in the presence of lease transfers, it is possible for some writes in a transaction that straddle multiple ranges to be lost. In cases where these lost writes are not also present on one or more secondary indexes that could allow for reconstructing or repairing the row, these writes could be irrecoverably lost.
 
-However, in certain scenarios, these writes may be fully recoverable. For example, if the lost writes were on a secondary index, these could be fully repaired from the primary index, or a primary index could be fully repaired if there exists a set of secondary indexes that fully covers the primary index columns. The lost writes may also be partially recoverable. For example, if only some of the lost writes from a primary index are contained in secondary indexes.
+Single-range writes, implicit transactions, and workloads using the One-Phase-Commit (1PC) fast path are not affected.
```

Comment:
Saying "implicit transactions" here is NOT correct. Let's remove it and just say:

"Transactions that write to a single range, or transactions that take advantage of the One-Phase-Commit (1PC) optimization, are not affected."

For context, saying "implicit transactions" is not correct, because you can have something like an `INSERT ... VALUES (a), (b);` that straddles multiple ranges, or simply an `INSERT ... VALUES (a)` that needs to update secondary indexes.